backend/clubs/views.py

```diff
@@ -4861,6 +4863,39 @@ def get_queryset(self):
         )
 
 
+class WhartonCyclesView(viewsets.ModelViewSet):
+    """
+    update: Update application cycle
+    list: Get list of all application cycles
+    """
+
+    permission_classes = [WhartonApplicationPermission | IsSuperuser]
+    http_method_names = ["get", "post", "put", "patch", "delete"]
+    serializer_class = ApplicationCycleSerializer
+
+    def get_queryset(self):
+        return ApplicationCycle.objects.all().order_by("end_date")
+
+    def list(self, request, *args, **kwargs):
+        return super().list(request, *args, **kwargs)
+
+    def update(self, *args, **kwargs):
+        """
+        Updates times for all applications with cycle
+        """
+        applications = ClubApplication.objects.filter(
+            application_cycle=self.get_object()
+        )
+        for app in applications:
+            app.application_start_time = self.get_object().start_date
+            app.application_end_time = self.get_object().end_date
+            if app.result_release_time < app.application_end_time:
+                app.result_release_time = app.application_end_time
+                +datetime.timedelta(days=10)
```

Comment:
Why +10 days always? Should we allow the user to pass in a field controlling the cycle's result release time?

Reply:
Was a placeholder since release_time isn't currently a field in application_cycle, in order to avoid validation errors in the [email serializer](https://github.com/pennlabs/penn-clubs/blob/8dae4d6527fee32731d33e9f08ca45a65c32914b/backend/clubs/serializers.py#L2672) , there's probably a better UI solution to maintain the different invariants plus adding release_time ofc